modules/rosa-sts-account-roles-terraform.adoc

```diff
@@ -0,0 +1,217 @@
+
+.Procedure
+
+. Check your AWS account for existing roles and policies by running the following command:
++
+[source,terminal]
+----
+$ rosa list account-roles
+----
+
+. In your terminal, run the following command to export link:https://console.redhat.com/openshift/token[your OCM token]. This value must include the full OCM token:
++
+[source,terminal]
+----
+export RHCS_TOKEN="<your_ocm_token>"
+----
+
+. Optional: run the following command to specify your own account-role prefix that prepends the roles you create by running the following command:
++
+[NOTE]
+====
+If you do not specify an account-role prefix, a prefix is generated in the format of `account-role-` followed by a string of four random characters.
+====
++
+[source,terminal]
+----
+export account_role_prefix=<account_role_prefix>
+----
+
+. Create the Terraform files locally by using the following code templates:
+.. The `main.tf` file calls the Red Hat Cloud Services Terraform provider, which allows you to use OpenShift services with Terraform. Run the following command to create the `main.tf` file:
++
+[source,terraform]
+----
+$ cat<<-EOF>main.tf 
+  #
+  # Copyright (c) 2022 Red Hat, Inc.
+  #
+  # Licensed under the Apache License, Version 2.0 (the "License");
+  # you may not use this file except in compliance with the License.
+  # You may obtain a copy of the License at
+  #
+  #   http://www.apache.org/licenses/LICENSE-2.0
+  #
+  # Unless required by applicable law or agreed to in writing, software
+  # distributed under the License is distributed on an "AS IS" BASIS,
+  # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  # See the License for the specific language governing permissions and
+  # limitations under the License.
+  #
+
+  terraform {
+    required_providers {
+      aws = {
+        source  = "hashicorp/aws"
+        version = ">= 4.20.0"
+      }
+      rhcs = {
+        version = ">= 1.3.0"
+        source  = "terraform-redhat/rhcs"
+      }
+    }
+  }
+
+  data "rhcs_policies" "all_policies" {}
+
+  data "rhcs_versions" "all" {}
+
+  module "create_account_roles" {
+    source  = "terraform-redhat/rosa-sts/aws"
+    version = "0.0.15"
+
+    create_operator_roles = false
+    create_oidc_provider  = false
+    create_account_roles  = true
+
+    account_role_prefix    = var.account_role_prefix
+    rosa_openshift_version = var.openshift_version
+    account_role_policies  = data.rhcs_policies.all_policies.account_role_policies
+    operator_role_policies = data.rhcs_policies.all_policies.operator_role_policies
+    all_versions           = data.rhcs_versions.all
+    tags                   = var.tags
+  }
+EOF
+----
+
+.. You define the account role prefix structure in the `output.tf` file. This output definition allows you to specify how the various generated roles are constructed. Running the following command creates your `output.tf` file:
```

Comment:
Just a personal opinion but I think the following suggestion sounds a bit better and more closely follows the wording you use in other steps: 

```suggestion
.. You define the account role prefix structure in the `output.tf` file. This output definition allows you to specify how the various generated roles are constructed. Run the following command to create your `output.tf` file:
```
Feel free to ignore if there's a particular reason you chose this wording here.